fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.fcrepo.auth.common;
 
-import static org.fcrepo.kernel.api.FedoraJcrTypes.JCR_CONTENT;
+import static org.modeshape.jcr.api.JcrConstants.JCR_CONTENT;
```

Comment:
The argument here makes sense. Can you also describe how this current PR moves us in the direction of the "parent" ticket:
https://jira.duraspace.org/browse/FCREPO-1694 : "*Ensure that no non-deployable module compile-depends on fcrepo-kernel-modeshape*"
...or maybe the intent/title of the above ticket is simply misleading.